components/match2/commons/match_group_input_util.lua

```diff
@@ -1377,13 +1385,28 @@ function MatchGroupInputUtil.calculatePlacementOfOpponents(opponents)
 	local lastScore
 	local lastPlacement = 0
 
-	local function scoreSorter(tbl, key1, key2)
+	local function sorter(tbl, key1, key2)
+		local opponent1 = tbl[key1]
+		local opponent2 = tbl[key2]
+
+		if opponent1.status == MatchGroupInputUtil.STATUS_INPUTS.DEFAULT_WIN then
+			return true
+		elseif Table.includes(MatchGroupInputUtil.STATUS_INPUTS, opponent1.status) then
```

Comment:
is `STATUS_INPUT` correct here? Shouldn't it be `STATUS`?

Reply:
i think you review an outdated version